src/cli/init.rs

```diff
@@ -31,13 +31,35 @@ pub enum ManifestFormat {
 }
 
 /// Creates a new workspace
+///
+/// It initializes a `pixi.toml` file and also prepares a `.gitignore`
+/// to prevent the environment from being added to `git`.
+///
+/// If you have a `pyproject.toml` file in the directory where you run `pixi init`,
+/// it appends the pixi data to the `pyproject.toml` file instead of creating a new `pixi.toml` file.
+///
+/// When importing an environment, the `pixi.toml` will be created with the dependencies from the environment file.
+/// The `pixi.lock` will be created when you install the environment.
+/// We don't support `git+` urls as dependencies for pip packages and for the `defaults` channel we use `main`,
+/// `r` and `msys2` as the default channels.
+///
+///     pixi init myproject
+///     pixi init ~/myproject
+///     pixi init  # Initializes directly in the current directory.
+///     pixi init --channel conda-forge --channel bioconda myproject
+///     pixi init --platform osx-64 --platform linux-64 myproject
+///     pixi init --import environment.yml
+///     pixi init --format pyproject
+///     pixi init --format pixi --scm gitlab
+///
 #[derive(Parser, Debug)]
+#[clap(verbatim_doc_comment)]
```

Comment:
we would need to use `verbatim_doc_comment` for every comment, so that `--help` show something readable. As the raw text will be displayed in the terminal we will need to decide how much markdown we accept in here.